### DISCUSSION TITLE
Multilingual: Missing dates and "By" author

### DISCUSSION THREAD

**Discussion by elrezad** [gohugo-ananke/ananke#448] [2 - Q&A / Support]
Hi all, thanks you for your great work.

I'm new to Hugo, but I've managed to have a multilingual (EN, IT & ES) blog. However, I'm having some problems with the multilingual features. It would be great if someone could give me a hint of how to proceed.

Specifically, I haven't been able to :

1. Translate the post dates. I've been able to change the format, but the month's names are still in english.
2. Translate the "By " author below the post title.

Anything that I'm missing?

Thanks again

**Comment by nicolasVr** [comment]
I agree, and it would be nice to be able to translate the "more" at the end of posts list.
